Gemfile

```diff
@@ -91,4 +91,4 @@ group :test do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
Bitte füge die `EditorConfig` wie vorgeschlagen zum Projekt hinzu und nutze diese.

Reply:
Ich bin von Windows to Linux gewechsel und dann das entfernt.

Reply:
`EditorConfig` brauchst du überall. Die gehört ins Repo.

Reply:
Die EditorConfig ist nicht default in Rails bei aber ich kann es hinzufügen.

Reply:
Das ist richtig. Hat auch nichts mit Rails zu tun, sondern mit unseren Qualitätsanforderungen an Projekt! Schau dazu die entsprechenden Folien an.